File: PHP/NIM.php
<?php
echo "Masukkan NIM: ";
$nim = trim(fgets(STDIN));

$digits = [];
for ($i = 0; $i < strlen($nim); $i++) {
    $digits[] = intval($nim[$i]);
}

$total = array_sum($digits);
$maks = max($digits);
$minim = min($digits);
$rata = $total / count($digits);

$reverse = array_reverse($digits);

echo "\nArray digit: ";
print_r($digits);

echo "Total digit: $total\n";
echo "Nilai maksimum: $maks\n";
echo "Nilai minimum: $minim\n";
echo "Rata-rata digit: $rata\n";

echo "Reverse array: ";
print_r($reverse);
?>

File: Python/nim.py
nim = input("Masukkan NIM: ")

digits = []
for c in nim:
    digits.append(int(c))

total = sum(digits)
maks = max(digits)
minim = min(digits)
rata = total / len(digits)

reverse_digits = digits[::-1]

print("Array digit:", digits)
print("Total digit:", total)
print("Nilai maksimum:", maks)
print("Nilai minimum:", minim)
print("Rata-rata digit:", rata)
print("Reverse array:", reverse_digits)
